packages/plugin-core/src/commands/DeleteNodeCommand.ts

```diff
@@ -32,6 +38,8 @@ function formatDeletedMsg({
   return `${path.basename(fsPath)} (${VaultUtils.getName(vault)}) deleted`;
 }
 
+const md = _md();
```

Comment:
can we load this inside of the command? in general, avoid having side effects on import (_md will also be initialized whenever delete node is imported)